pkg/server/server.go

```diff
@@ -36,3 +42,12 @@ var Module = fx.Options(
 		scheduler.NewScheduler,
 	),
 )
+
+func provideNIl() (prometheus.Registerer, logr.Logger) {
```

Comment:
what is the `NIl` means?

Reply:
would I rename it to provide()?

Reply:
It is updated in previous pr, just ignore it here